configure.ml

```diff
@@ -512,6 +512,8 @@ let camltag = match caml_version_list with
   | x::y::_ -> "OCAML"^x^y
   | _ -> assert false
 
+let coq_warn_flag =
+  if caml_version_nums >= [4;2;0] then "-w -3-52-56" else ""
```

Comment:
If the `capitalize` warning is the single one, indeed you want Gaetan's selector.